src/index.js

```diff
@@ -72,6 +77,19 @@ export default function fetch (url, opts = {}) {
     }
     let reqTimeout
 
+    const abortRequest = () => {
+      reject(new FetchError('request aborted', 'abort'))
+      if (request.useElectronNet) {
+        req.abort()
+      } else {
+        req.destroy(new FetchError('request aborted', 'abort'))
```

Comment:
Good catch, using `destroy` instead of `abort` in node case, since `abort` is deprecated. We should do the same in the timeout handler